### PR TITLE
Changelog for v2.1.1 Release

### PR DESCRIPTION
**Pull request by priyanka1110**
### Goal
CHANGELOG for v2.1.1 release.

### Implementation
- Updated CHANGELOG for v2.1.1 release.
- Fixed the broken changelog.

### CODE REVIEW

**Review by rahulkarkera — APPROVED** [review]
LGTM

**Review by prabhumarappan — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
+
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+
+## [Unreleased]
+### Changed
+- Contact information and bangalore office address has been updated in the contact page.
+
+### Removed
+- Infratab.com access from footer has been removed.
```
Comment (moosicalbovine) [inline CHANGELOG.md]: ~access~ -> link

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,22 @@
...
+# Changelog
 All notable changes to this project will be documented in this file.
-s
+
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+
+## [Unreleased]
+### Changed
+- Contact information and bangalore office address has been updated in the contact page.
```
Comment (moosicalbovine) [inline CHANGELOG.md]: ~has~ -> have

**Review by moosicalbovine — CHANGES_REQUESTED** [review]

**Comment by priyanka1110** [comment]
@moosicalbovine I have updated the changelog as per the comments.

File: CHANGELOG.md [hunk 3]
```diff
@@ -17,29 +28,27 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
...
 ### Added
-- More award has been added in award section
-- Real map has been added instead of map image in contact page
-- Form validation has been
-
-### Updated
-- Life at infratab section image layout has been updated
+- More award has been added in award section.
+- Real map has been added instead of map image in contact page.
+- Form validation has been.
```
Comment (niren777) [inline CHANGELOG.md]: @priyanka1110 @moosicalbovine the sentence is not completed

Comment (priyanka1110) [reply CHANGELOG.md]: Ok, @niren777 Will update it.

**Review by niren777 — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,11 +1,22 @@
...
+# Changelog
 All notable changes to this project will be documented in this file.
-s
+
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+
+## [Unreleased]
+### Changed
+- Contact information and bangalore office address have been updated in the contact page.
```
Comment (moosicalbovine) [inline CHANGELOG.md]: - ~bangalore~ -> "Bangalore"
- Contact should be capitalised too

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,11 +1,22 @@
...
+
+### Removed
+- Infratab.com link from footer has been removed.
```
Comment (moosicalbovine) [inline CHANGELOG.md]: ~link~ -> hyperlink/URL

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,11 +1,22 @@
...
+
+### Fixed
+- Contact page title typo has been fixed.
```
Comment (moosicalbovine) [inline CHANGELOG.md]: - The typo in the Contact page's title is fixed

**Review by moosicalbovine — COMMENTED** [review]
Please address my in-line comments

File: CHANGELOG.md (hunk 4 above)
Comment (priyanka1110) [reply CHANGELOG.md]: @moosicalbovine I have updated the changes in the changelog as per the comments.

File: CHANGELOG.md (hunk 5 above)
Comment (priyanka1110) [reply CHANGELOG.md]: @moosicalbovine I have updated the changes in the changelog as per the comment.

File: CHANGELOG.md (hunk 6 above)
Comment (priyanka1110) [reply CHANGELOG.md]: @moosicalbovine I have updated the changes in the changelog as per the comment.

**Comment by priyanka1110** [comment]
@moosicalbovine I address all your inline comments and removed the full stops from the changelog as per earlier instructions in the other repo.

**Pull request merged**